Firestore/Swift/Tests/Codable/CodableGeoPointTests.swift

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2018 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import FirebaseFirestore
+import FirebaseFirestoreSwift
+import Foundation
+import XCTest
+
+class CodableGeoPointTests: XCTestCase {
+  func testGeoPointEncodes() {
+    let geoPoint = GeoPoint(latitude: 37.77493, longitude: -122.41942)
+
+    let jsonData = try! JSONEncoder().encode(geoPoint)
+    let json = String(data: jsonData, encoding: .utf8)!
+
+    // The ordering of attributes in the JSON output is not guaranteed, nor is the rounding of
```

Comment:
"`nor is the rounding of the values`"

Is this going to be an issue do you think? Or is it just .0001 off like below?

Reply:
The issue here is that we don't control the JSON encoding/decoding and this test doesn't really care about any kind of conformance in that scenario. If users use this serialization and don't like the results they can take it up with the authors of `JSONEncoder`.

I'm validating that GeoPoint conforms to Codable without any dependency on Firestore or snapshots.